pbsync/pbsync.py

```diff
@@ -81,7 +81,9 @@ def sync_handler(sync_val: str, repository_val=None, requested_bundle_name=None)
                 "Unreal Editor is currently running. Please close it before running PBSync. It may be listed only in Task Manager as a background process. As a last resort, you should log off and log in again.")
 
         pblog.info("Fetching recent changes on the repository...")
-        subprocess.call(["git", "fetch", "origin"])
+        subprocess.run(["git", "fetch", "origin", "promoted"])
```

Comment:
Instead of fetching those 3 branches, we should fetch only the branch we're on. Creatives should not be fetching stuff in trunk & master imo.

Reply:
UE4GitPlugin now requires this behavior, so that we can get remote information from all relevant branches, to check if there is a newer file on another branch.

Reply:
https://github.com/ProjectBorealis/UE4GitPlugin/blob/pb-changes/Source/GitSourceControl/Private/GitSourceControlUtils.cpp#L1032